app/jobs/cache/record_counts/recent_additions_job.rb

```diff
@@ -0,0 +1,47 @@
+module Cache
+  module RecordCounts
+    class RecentAdditionsJob < Cache::RecordCountsJob
+      def perform
```

Comment:
Assignment Branch Condition size for perform is too high. [40.27/15]<br>Method has too many lines. [29/10]